package.json

```diff
@@ -17,6 +17,7 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@types/node": "^20.10.6",
```

Comment:
```suggestion
```

Could you mention why was this dependency added?

Reply:
I was working with react and your project is on TypeScript so i tried to configure, in between that process i did add  those dependency and in the end forget to remove it.